### DISCUSSION TITLE
Warning: Undefined array key "en" when creating a public share [PORTAL ENGINE]

### DISCUSSION THREAD

**Discussion by Yuki1602** [pimcore/pimcore#10840] [Q&A / Support]
getting the error
 An exception occurred while executing 'INSERT INTO portal_engine_public_share (collectionId, hash, name, userId, currentSiteId, expiryDate, showTermsText, termsText, configs, creationDate, modificationDate) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [null, "54dae8fafe26e6dabbdaf1b267f8a594", "dsadsad", 1797, 2, 1638489600, 0, "", "{\"51\":[{\"type\":\"structuredData\",\"localized\":false,\"attribute\":null,\"format\":\"Pimcore\\\\Bundle\\\\PortalEngineBundle\\\\Service\\\\DataPool\\\\DownloadFormat\\\\CsvDownloadFormat\",\"setup\":null},{\"type\":\"asset\",\"localized\":false,\"attribute\":null,\"format\":\"original\",\"setup\":null}]}", 1637046315, 1637046315]: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'collectionId' cannot be null

when creating a single element public share and when creating a collection public share i get the error in the title

**Comment by Yuki1602** [comment]
any answers?

**Reply by fashxp** [reply]
I guess there is something wrong with your database table ... for me, `collectionId` is nullable ... and it also needs to be... 

![image](https://user-images.githubusercontent.com/8792145/144386913-61382d4a-01a8-48be-a48d-7b9b799f6220.png)
